pachyderm/values.yaml

```diff
@@ -34,6 +34,14 @@ dash:
     #requests:
     #  cpu: "1"
     #  memory: "2G"
+  config:
+    issuerURI: "http://localhost:30658/"
```

Comment:
These values should be moved into `examples/dash-local.yaml` (You'll need to leave blank values here for json schema)

Reply:
You can leave graphqlPort and pachdAddress